### ISSUE TITLE
为什么Conv2D的结果每次都不一样

### ISSUE THREAD

**Issue by xiaomujiang** [PaddlePaddle/Paddle#28209]
使用下面的test模型，在aistudio测试Conv2D的结果，每次运行的时候，得到结果都是不一样，这是哪里写错了吗？还是有什么随机数的步骤
import numpy as np
import paddle.fluid as fluid
from paddle.fluid.dygraph import Conv2D
from paddle.fluid.dygraph import to_variable
class Test(fluid.dygraph.Layer):
    def __init__(self,name_scope):
        super(Test,self).__init__(name_scope)
        self.conv = Conv2D(num_channels=1,num_filters=1,filter_size=3,stride=1,padding=1)
    def forward(self,inputs):
        x = self.conv(inputs)
        return x

if __name__ == "__main__":
    use_cuda = False
    place = fluid.CUDAPlace(0) if use_cuda else fluid.CPUPlace()
    with fluid.dygraph.guard(place):
        model = Test("test")
        model.eval()
        input_data = np.array([[1,2,3],[4,5,6],[7,8,9]]).astype(np.float32)
        input_data=input_data[np.newaxis,np.newaxis,:,:]
        input_data = to_variable(input_data)
        output_data = model(input_data)
        output_data = output_data.numpy()
        print('Output data shape: ', output_data.shape,output_data)

运行的结果

aistudio@jupyter-172898-1138028:~/work$ python tet.py 
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/sklearn/externals/joblib/externals/cloudpickle/cloudpickle.py:47: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
Output data shape:  (1, 1, 3, 3) [[[[1.486265  2.1052248 2.7016432]
   [2.8810153 3.0953333 6.040648 ]
   [3.148071  5.5301747 7.718881 ]]]]
aistudio@jupyter-172898-1138028:~/work$ python tet.py 
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/sklearn/externals/joblib/externals/cloudpickle/cloudpickle.py:47: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
Output data shape:  (1, 1, 3, 3) [[[[-0.9979699  1.7792698  3.7427025]
   [-1.9500908  3.1212916  5.5841837]
   [-2.5594819 -2.1905074 -0.8402663]]]]
aistudio@jupyter-172898-1138028:~/work$ python tet.py 
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/sklearn/externals/joblib/externals/cloudpickle/cloudpickle.py:47: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
Output data shape:  (1, 1, 3, 3) [[[[-3.0148313  -0.69926673 -1.7474635 ]
   [-6.255338   -1.0932407  -3.7442732 ]
   [-2.8909595  -2.326908   -6.8422623 ]]]]

**Comment by DannyIsFunny** [comment]
因为这个新建的Conv2D，没有指定weight和bias信息。 其weight和bias信息使用了随机值。
https://www.paddlepaddle.org.cn/documentation/docs/zh/api_cn/dygraph_cn/Conv2D_cn.html#conv2d
![image](https://user-images.githubusercontent.com/45189361/96876422-058caa80-14ab-11eb-9b45-acf7bc78f74a.png)

每一次计算都是用了一个新的随机的weight和bias，导致Conv2D在同样输入下输出不同
